package.json

```diff
@@ -81,7 +86,9 @@
       "react/prop-types": 0,
       "react/react-in-jsx-scope": 0,
       "standard/computed-property-even-spacing": 0,
-      "@emotion/pkg-renaming": 2
+      "@emotion/pkg-renaming": 2,
+      "no-use-before-define": 0,
+      "@typescript-eslint/no-use-before-define": 1
```

Comment:
interesting - isn't it one of the goals of TypeScript ESLint project to hide this from consumers? 🤔 I mean - don't they automatically switch to TS-oriented variants of plugins like this?

Reply:
I think we might need to enable the `recommended` rules from `@typescript-eslint/eslint-config` for that. I didn't want to enable it since I wasn't sure how well it'd play together with `standard`. Since we're dropping that, I'll enable the recommended rules and see how that goes.

Reply:
Yeap, `extends:@typescript-eslint/eslint-recommended" removed the need for this.